modules/@apostrophecms/schema/ui/apos/components/AposInputArea.vue

```diff
@@ -66,6 +62,17 @@ export default {
     }
   },
   methods: {
+    getEmptyValue() {
+      return {
+        metaType: 'area',
+        _id: cuid(),
+        items: []
+      };
+    },
+    watchValue () {
+      this.error = this.value.error;
+      this.next = this.value.data || this.getEmptyValue();
```

Comment:
Makes this component more robust by tolerating being switched to a `null` value later just as we tolerate it at startup.

Reply:
I tried to reproduce the bug for array fields, which have a similar thing going on, but could not, so I kept the fix minimal.